test/Microsoft.Azure.SignalR.Protocols.Tests/ServiceProtocolFacts.cs

```diff
@@ -68,5 +223,35 @@ private static byte[] Frame(byte[] input)
                 MemoryBufferWriter.Return(stream);
             }
         }
+
+        private static ServiceMessage ParseServiceMessage(byte[] bytes)
```

Comment:
Why do we need this method?

Reply:
Used [here](https://github.com/Azure/azure-signalr/pull/105/files#diff-6dee76e558c2ed86e6135a0ebd3ece52R157) because we don;t have the specific type.
Making it a method to look nicer.

Reply:
Can't you call the generic one with ServiceMessage as the type?

Reply:
Yeah, i tried but failed with error. Looks like `IsType` will check the exact type.